sdk/samples/iot/README.md

```diff
@@ -169,6 +166,267 @@ To run the samples, ensure you have the following programs and tools installed o
     git clone https://github.com/Azure/azure-sdk-for-c.git
     ```
 
+## Getting Started
+
+Next you must create and connect a device. You can do this in one of two ways: via Azure IoT Hub or via Azure IoT Hub Device Provisioning Service (DPS). You also must choose how to authenticate the device, either via X.509 Self-Signed Certification or Symmetric Key (SAS).
+
+### Create a Device Using X.509 Self-Signed Certification
+
+This approach must be used for the following samples: `paho_iot_hub_c2d_sample`, `paho_iot_hub_methods_sample`, `paho_iot_hub_telemetry_sample`, `paho_iot_hub_twin_sample`, `paho_iot_hub_pnp_sample`, `paho_iot_hub_pnp_component_sample`, `paho_iot_provisioning_sample`
+
+1. Generate a certificate
+
+   As a convenience, we provide a series of commands below for you to create a temporary certificate in order to run the samples. These certificates expire after one day and are provided ONLY to help you easily understand CA Certificates. When productizing against CA Certificates, you will need to use your own security best practices for certificate creation and lifetime management.
+
+    **WARNING: Certificates created by these commands MUST NOT be used in production-level code on Windows or macOS.**
```

Comment:
```suggestion
    **WARNING: Certificates created by these commands MUST NOT be used in production-level code.**
```
probably want to prevent people thinking "hey cool alright I'll use it on my microcontroller or on Linux"